app/views/contacts/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="container mt-3">
+  <div class="row justify-content-center">
+    <div class="col-12" %>
+      <h1>New Contact</h1>
+
+      <%= render 'form', contact: @contact %>
+
+</div>
```

Comment:
@jackmiddleton same as above: always check (or even count) your closing divs, you're miss a few of these here